FF1Lib/FF1Rom.cs

```diff
@@ -290,6 +290,8 @@ public void Randomize(Blob seed, Flags flags, Preferences preferences)
 				ShuffleUnrunnable(rng);
 			}
 
+			CompletelyUnrunnable();

```

Comment:
You'd better check your flag here chief - and Ideally I'd actually put this inside the unrunnable shuffle block to be double obvious. 

You have to check Unrunnable Shuffle and THEN check off Completely unrunnable....

But maybe this should even be scalable? So you can have half the number of unrunnables... or double.... or 0% or 100%.... just dreaming this is a nice start.